Tests/SystemTests/PublisherAuthenticationSystemTests.swift

```diff
@@ -121,7 +121,7 @@ class PublisherAuthenticationSystemTests: XCTestCase {
             .connection(configuration)
             .mapboxConfiguration(MapboxConfiguration(mapboxKey: Secrets.mapboxAccessToken))
             .log(logConfiguration)
-            .locationSource(LocationSource(locationSource: [CLLocation(latitude: 0.0, longitude: 0.0)]))
+            .locationSource(LocationSource(locationSource: [CLLocation(latitude: 0.0, longitude: 0.0), CLLocation(latitude: 1.0, longitude: 1.0)]))
```

Comment:
Do we want to revert this once Mapbox release version 2.1 of their SDK? If so, we should create an issue to track this.

Reply:
No, we won't revert it, this is totally fine for testing purposes.